.github/workflows/ci-publish.yaml

```diff
@@ -0,0 +1,47 @@
+name: "Publish Docs"
+
+on:
+  workflow_run:
+    workflows: ["CI"]
+    branches: [master]
+    types: 
+      - completed
+
+jobs:
+  publishing:
+    name: Publish Documentation
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+    - name: 📥 Download stable docs with released
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: release.yaml
+        workflow_conclusion: success
+        name: stable
+        path: docs/public/
```

Comment:
Question: how can you be sure from which workflow run your fetching artifacts? I didn't find any documentation in the GitHub action as to which policy it uses to select between several run. Did you find something about it?